businesscentral-runtime/image.yaml

```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-businesscentral-runtime"
+description: "Red Hat JBoss BPM Suite 7.0 Business Central Runtime Views container image"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "bpmsuite-businesscentral-runtime"
+    - name: "JBOSS_BPMSUITE_BUSINESSCENTRAL_VERSION"
+      value: "7.0.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.0"
+cmd:
+    - "/opt/eap/bin/standalone.sh"
+    - "-b"
+    - "0.0.0.0"
+    - "-c"
+    - "standalone.xml"
+scripts:
+    - package: businesscentral-runtime
+      exec: install
+sources:
+    - artifact: jboss-bpmsuite-7.0.0.redhat-20170824.071756-1-business-central-runtime-eap7.zip
```

Comment:
This artifact name is an initial approach. It will probably change.

Reply:
jboss-bpmsuite-7.0.0-business-central-monitoring-eap7.zip